resources/views/layout.blade.php

```diff
@@ -94,7 +94,7 @@
                     </li>
                 </ul>
 
-                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                @if(Auth::check() && Session::has('auth.vatsim-sso'))
```

Comment:
Made a couple changes, now the main account panel will be displayed if you're fully authenticated, but if only through Vatsim SSO then just a logout button shows. This avoids the call to null that was occurring previously.